File: nilai.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Masukkan nilal tugas: ");
    double tugas = sc.nextDouble();
    System.out.print("Masukkan nilai UTS: ");

    double uts= sc.nextDouble();
    System.out.print("Masukkan nilai UAS: ")
    double uas = sc.nextDouble();

    double nilaiAkhir = (0.3* tugas) + (0.3* uts) + (0.4* uas);

    System.out.println("Nilai Akhir = " + nilaiAkhir);
    if (nilaiAkhir >= 60)
    System.out.println("Status: LULUS");
    else
    System.out.println("Status: TIDAK LULUS");
    {

}

File: nilai.php
<?php
$tugas = readline("Masukkan nilai tugas: ");
$uts = readline("Masukkan nilai UTS: ")
$uas = readline("Masukkan nilai UAS: ");

$nilalAkhir = (0.3 $tugas) + (0.3 $uts) + (0.4* $uas);

echo "Nilai Akhir = $nilaiAkhir\n";
if ($nilaiAkhir >= 60)
echo "Status: LULUS\n";
else
echo "Status: TIDAK LULUS\n";
?>

